tests/e2e/default/enketo/edit-report-with-attachment.wdio-spec.js

```diff
@@ -65,14 +64,11 @@ describe('Edit report with attachmnet', () => {
     await utils.seedTestData(userContactDoc, [...docs, formDoc]);
     await loginPage.cookieLogin();
     await common.hideSnackbar();
-  });
-
-  it('should remove attachment when saving', async () => {
     reportDoc._id = uuid();
     await utils.saveDoc(reportDoc);
+  });
 
-    await commonElements.goToReports();
```

Comment:
How do you ensure you are on the reports tab?

Reply:
Because in the method `editReport` from the `reports.wdio.page` file it goes to the report's tab 🙂 

![image](https://github.com/medic/cht-core/assets/94494491/4c589b7d-346b-407e-b802-c1511adb91e9)